package.json

```diff
@@ -23,12 +23,14 @@
     "ts-node": "^10.9.1"
   },
   "devDependencies": {
+    "@cdklabs/cdk-enterprise-iac": "^0.0.303",
```

Comment:
Is this needed since `osml-cdk-constructs` already have this package?

Reply:
Yes, the build fails without it